pkgs/by-name/co/construct/package.nix

```
@@ -0,0 +1,37 @@
+{ llvmPackages
+, stdenv
+, lib
+, fetchFromGitHub
+, boost
+}:
+
+stdenv.mkDerivation rec {
```

From what I've picked up `finalAttrs` is preferred over `rec`, maybe someone more familiar with the details could chime in and provide some background information in this?

```suggestion
stdenv.mkDerivation (finalAttrs: {
```